example/bond/package.json

```diff
@@ -14,6 +14,7 @@
   "devDependencies": {
     "@types/node": "^22.8.7",
     "copy-file": "^11.0.0",
+    "ts-node": "^10.9.2",
```

Comment:
This does seem to be needed

Reply:
Added to https://github.com/LF-Decentralized-Trust-labs/paladin/pull/389 as well.